build_library/prod_image_util.sh

```diff
@@ -102,7 +103,10 @@ create_prod_image() {
   fi
 
   tar -cf "${BUILD_DIR}/${image_pkgdb}" -C "${root_fs_dir}" var/cache/edb var/db/pkg
-
+  sudo cp -r "${root_fs_dir}" "${BUILD_DIR}/root_fs_dir2" 
+  sudo rsync -a --delete  "${BUILD_DIR}/configroot/etc/portage" "${BUILD_DIR}/root_fs_dir2/etc" 
+  mksquashfs "${BUILD_DIR}/root_fs_dir2"  "${BUILD_DIR}/flatcar_production_image_sysext.squashfs" -noappend
+  
```

Comment:
As test case, how about running the script here to create a python or docker sysext? Then we can also see it pass from the GitHub Actions CI workflow.

Reply:
(Storing a docker sysext on the image is out of scope for this PR but I think it would be good to have the script used somewhere to see that it works and whether it breaks)

Reply:
Yes, thanks